Backend/app/provider-platform/static-offer-driver-app/Main/src/Domain/Action/Dashboard/Ride.hs

```diff
@@ -159,9 +159,9 @@ rideInfo merchantShortId reqRideId = do
         driverPhoneNo,
         vehicleNo = rideDetails.vehicleNumber,
         driverStartLocation = ride.tripStartPos,
-        driverCurrentLocation = getCoordinates driverLocation,
+        driverCurrentLocation = Just $ getCoordinates driverLocation,
```

Comment:
why you did not applied the same here? 
`driverCurrentLocation = getCoordinates <$> mDriverLocation,`

Reply:
static-offer-driver-app we will be deprecating